lambdas/search/index.py

```diff
@@ -23,6 +23,63 @@
 README_KEYS = ['README.md', 'README.txt', 'README.ipynb']
 SUMMARIZE_KEY = 'quilt_summarize.json'
 
+DELETED_METRIC = {
+    "scripted_metric": {
+        "init_script": """
+            state.last_modified = 0;
+            state.deleted = false;
+        """,
+        "map_script": """
+            def last_modified = doc.last_modified.getValue().toInstant().toEpochMilli();
+            if (last_modified > state.last_modified) {
+                state.last_modified = last_modified;
+                state.deleted = doc.delete_marker.getValue();
+            }
+        """,
+        "reduce_script": """
+            def last_modified = 0;
+            def deleted = false;
+            for (s in states) {
+                if (s.last_modified > last_modified) {
+                    last_modified = s.last_modified;
+                    deleted = s.deleted;
+                }
+            }
+            return deleted ? 1 : 0;
+        """,
+    },
+}
+
+
+def make_sample_objects_agg(num):
+    return {
+        'terms': {
+            'field': 'key',
+            'order': [{'modified': 'desc'}],
+            'size': 1000000,
```

Comment:
Can you add a comment on this? It's way more than the JS client could handle? If we're trying to limit server-side work for performance reasons, tweaking MAX_DOCUMENTS_PER_SHARD will probably be faster.
```suggestion
            'size': 1000000,
```